tiledb/cloud/tiledb_cloud_error.py

```diff
@@ -12,7 +12,8 @@ def check_exc(exc):
         "[InternalError: failed to parse or message missing from ApiException]"
     )
 
-    if not isinstance(exc, BaseException):
+    # Make sure exc.status and exc.body exist before dereferncing them.
+    if not isinstance(exc, ApiException):
```

Comment:
ApiException is not a defined name; you need to

```python
from tiledb.cloud import rest_api

# ...

if not isinstance(exc, rest_api.ApiException):
  # ...
```

Reply:
> It’s very difficult to follow.

Thanks @thetorpedodog ! I'll abandon and re-submit. The rebasing made things less clear (if you look one commit at a time).

Reply:
I.e. I see your confusion looking at https://github.com/TileDB-Inc/TileDB-Cloud-Py/pull/175/commits.

Looking at https://github.com/TileDB-Inc/TileDB-Cloud-Py/pull/175/files this doesn't look complex to me.

Nonetheless I'll abandon and re-submit a clean copy. Thanks for the feedback!